CONTRIBUTING.md

```diff
@@ -0,0 +1,126 @@
+
+To contribute a patch, the workflow is as follows:
+
+- Fork repository
+- Create topic branch
+- Commit patch
+- Create pull-request, adhering to the coding conventions herein set forth
+
+In general a commit serves a single purpose and diffs should be easily comprehensible. For this reason do not mix any formatting fixes or code moves with actual code changes.
+
+### Automated Test coverage
+
+All code submissions must be accompanied by appropriate automated tests. The goal is to provide confidence in the code’s robustness, while avoiding redundant tests.
+
+### Pull Requests
+
+The process described here has several goals:
+
+- Maintain Product quality
+- Fix problems that are important to users
+- Engage the community in working toward the best possible product
+- Enable a sustainable system for maintainers to review contributions
+- Further explanation on PR & commit messages can be found in this post: [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/).
+
+Please follow these steps to have your contribution considered by the approvers:
+
+1. Complete the CLA, as described in [CLA.md]
+2. Follow all instructions in [PULL-REQUEST-TEMPLATE.md](.github/pull_request_template.md)
+3. Include appropriate test coverage. Testing is 100% automated. There is no such thing as a manual test.
+4. Follow the [Style Guides](#style-guides)
+5. After you submit your pull request, verify that all [status checks](https://help.github.com/articles/about-status-checks/) are passing <details><summary>What if the status checks are failing?</summary>If a status check is failing, and you believe that the failure is unrelated to your change, please leave a comment on the pull request explaining why you believe the failure is unrelated. A maintainer will re-run the status check for you. If we conclude that the failure was a false positive, then we will open an issue to track that problem with our status check suite.</details>
+
+While the prerequisites above must be satisfied prior to having your pull request reviewed, the reviewer(s) may ask you to complete additional design work, tests, or other changes before your pull request can be ultimately accepted. Please refer to [Code Reviews].
+
+## Reporting Bugs
+
+Following these guidelines helps maintainers and the community understand your report, reproduce the behavior, and find related reports.
+
+Explain the problem and include additional details to help maintainers reproduce the problem:
+
+- **Use a clear and descriptive title** for the issue to identify the problem.
+- **Describe the exact steps which reproduce the problem** in as many details as possible. For example, start by explaining how you started Teku, e.g. which command exactly you used in the terminal, or how you started it otherwise.
```

Comment:
Replace "Teku" by "Orchestrate Node SDK" or just Orchestrate, I don't really know but not Teku ;)